mlrun/projects/pipelines.py

```diff
@@ -404,14 +404,25 @@ class _PipelineRunStatus:
     """pipeline run result (status)"""
 
     def __init__(
-        self, run_id, engine, project, workflow=None, state="", run_object=None
```

Comment:
note that this class is being returned to the user and user might expect run_obj, you are more familiar than me with this area, is this something the user might access?

Reply:
`run_object` is redundant and for inner use only, and because we don't use it anymore it can be removed